backwardcompatibilityml/tensorflow/helpers.py

```diff
@@ -2,6 +2,44 @@
 
 
 def bc_fit(h2, training_set=None, testing_set=None, epochs=None, bc_loss=None, optimizer=None):
+    """
+    This function is used to train a model h2, using an instance of a Tensorflow BCLoss function
+    that has been instantiated using an existing model h1 and regularization parameter lambda_c.
+
+    Example usage:
+
+        h2 = tf.keras.models.Sequential([
+          tf.keras.layers.Flatten(input_shape=(28, 28, 1)),
+          tf.keras.layers.Dense(128,activation='relu'),
+          tf.keras.layers.Dense(10, activation='softmax')
+        ])
+
+        lambda_c = 0.9
+        h1.trainable = False
+        bc_loss = BCCrossEntropyLoss(h1, h2, lambda_c)
+
+        optimizer = tf.keras.optimizers.Adam(0.001)
+
+        tf_helpers.bc_fit(
+            h2,
+            training_set=ds_train,
+            testing_set=ds_test,
+            epochs=6,
+            bc_loss=bc_loss,
+            optimizer=optimizer)
+
+    Args:
+        h2: A Tensorflow model that we want to train using backward compatibility.
+        training_set: The training set to use to train our model.
+        testing_set: The testing set to use to validate our model.
```

Comment:
Change to "The testing set for validating our model."

Reply:
Done.